integration-tests/jpa-derby/pom.xml

```
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-integration-tests-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-integration-test-jpa-derby</artifactId>
+    <name>Quarkus - Integration Tests - JPA - Derby</name>
+    <description>Module that contains JPA related tests running in strict mode with the Derby database</description>
```

```suggestion
    <description>Module that contains JPA related tests running with the Derby database</description>
```
This notion of `strict mode` is long gone, I will update the descriptions of the drivers.